src/node_options.h

```diff
@@ -306,6 +306,7 @@ class PerProcessOptions : public Options {
   bool report_compact = false;
   std::string report_directory;
   std::string report_filename;
+  bool report_disable_network = false;
```

Comment:
This should be in `EnvironmentOptions`, it would be rather weird if a worker can toggle the setting for the entire process.

Reply:
Just for my understanding, what is the difference between this option and the other `boolean` based ones like `compact` and `onfatalerror`?

Reply:
onfatalerror is a trigger, not a configuration in how report should look like. I think compact being per-process is a mistake. Think the following situation:

```
process.report.compact = true;
process.writeReport();  // Some other thread toggling process.report.compact to false
```

There would be a race unless users lock the report generation ability somehow.

Reply:
Makes sense! Fixed!